src/components/connection_handler/src/connection_handler_impl.cc

```diff
@@ -466,14 +467,6 @@ void ConnectionHandlerImpl::OnSessionStartedCallback(
     const uint32_t session_key =
         KeyFromPair(connection_handle, context.new_session_id_);
 
-    uint32_t app_id = 0;
```

Comment:
Should these changes be included in one of the #2190 or #2191 bug fix branches instead?